packages/nitro-rpc-client/scripts/client-runner.ts

```diff
@@ -183,12 +194,53 @@ yargs(hideBin(process.argv))
         closeCount++;
       }
 
-      aliceClient.Close();
-      ireneClient.Close();
-      bobClient.Close();
+      await closeClients(clients);
       process.exit(0);
     }
   )
+  .command(
+    "create-ledger",
+    "Create a ledger channel",
+    (yargsBuilder) => {
+      return yargsBuilder
+        .option("left", {
+          describe: "Left rpc node",
+          type: "string",
+          default: "alice",
+        })
+        .option("right", {
+          describe: "Right rpc node",
+          type: "string",
+          default: "irene",
+        });
+    },
+    async (yargs) => {
+      const { left, right } = yargs;
+      if (!isValidClientName(left) || !isValidClientName(right)) {
+        throw new Error("Invalid client name");
+      }
+
+      const clients = await initializeClients();
+      const leftClient = clients.get(left);
+      const rightClient = clients.get(right);
+      if (!leftClient || !rightClient) {
+        throw new Error("A client is undefined");
+      }
+      const rightAddress = await rightClient.GetAddress();
+
+      if (yargs.printnotifications) {
+        [leftClient, rightClient].forEach(logOutChannelUpdates);
+      }
+
+      // Setup ledger channels
+      console.log("Constructing ledger channels");
+      const ledger = await leftClient.DirectFund(rightAddress);
+      console.log(`Ledger channel ${ledger.ChannelId} created`);
+
+      await wait(1000);
```

Comment:
Couldn't we do something like 

```ts
      await leftClient.WaitForObjective(ledger.Id);
      await rightClient.WaitForObjective(ledger.Id);
```
Instead of waiting 1s?

Reply:
The approach you suggest is nice. There is a subtle race condition:
1. `DirectFund` method is called.
2. Objective completes.
3. `WaitForObjective` is called.

Filing https://github.com/statechannels/nitro-gui/issues/74 to keep track of this.